polygon-daemon/templates/deployment.yaml

```diff
@@ -0,0 +1,66 @@
+{{- range $part := .Values.parts }}
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ $part.name }}
+  labels:
+    app: {{ $part.label}}
+spec:
+  replicas: {{ $.Values.replicas }}
+  strategy:
+    type: Recreate
+  selector:
+    matchLabels:
+      app: {{ $part.label }}
+  template:
+    metadata:
+      labels:
+        app: {{ $part.label }}
+    spec:
+      containers:
```

Comment:
Should probably also include resource request sizes based on upstream recommendations (defined in values so they can be overridden).